src/py21cmfast/plotting.py

```diff
@@ -114,7 +114,7 @@ def coeval_sliceplot(
     Parameters
     ----------
     struct : :class:`~outputs._OutputStruct` or :class:`~wrapper.Coeval` instance
-        The output of a function such as `ionize_box` (a class containing several quantities), or
+        The output of a function such as `ionize_box` (a class containing several lightcones), or
```

Comment:
Is this a typo? output shouldn't be containing several lightcones

Reply:
Haha it's an IDE-typo. I did a change of a variable called "quantities" to "lightcones" and my IDE also looks for references in docstrings and comments. Unfortunately "quantities" is a word used for more than just a variable name.